Katello/katello
# Fixes #35668 - Show both applicable and installable errata on overview card

#### What are the changes introduced in this pull request?

Add a toggle group "Applicable / Installable" (with tooltips) to the errata overview card.
Add an empty state for when a host has unknown errata status.
Update the ErrataTab toggle group with the tooltips as well.

![errata-card-new-message](https://user-images.githubusercontent.com/22042343/198050129-7b075d2b-8625-409b-a6e3-12be1bcbda8a.png)
![errata-tab-toggle](https://user-images.githubusercontent.com/22042343/197841699-352736cc-1858-4e82-8c39-b03c2b45c9bd.png)
![unknown-errata-status](https://user-images.githubusercontent.com/22042343/197841702-280015ca-1657-46ee-8070-819680ac6b45.png)



#### Considerations taken when implementing this change?

We don't currently store applicable errata counts in the DB for each host.  I decided to keep it that way; applicable errata are calculated on demand in the rabl instead of adding new columns to the database.

#### What are the testing steps for this pull request?

1. Get a host with differing numbers of applicable and installable errata
2. View the overview card
3. Verify that when you click on a link, the resulting page is filtered to "All" errata when the toggle group is on "Applicable," and "Installable" when on "Installable."
4. Also view the new tooltips on the Errata page
